src/libsync/networkjobs.cpp

```diff
@@ -51,10 +51,6 @@ RequestEtagJob::RequestEtagJob(AccountPtr account, const QUrl &rootUrl, const QS
 {
     setProperties({ QByteArrayLiteral("getetag") });
     connect(this, &PropfindJob::directoryListingIterated, this, [this](const QString &, const QMap<QString, QString> &value) {
-        if (reply()->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt() != 207) {
```

Comment:
Handled in https://github.com/owncloud/client/blob/2639a9a79b1c350f95308ef8c672ef9538941d8c/src/libsync/networkjobs.cpp#L302-L302